zebra/zebra_router.c

```diff
@@ -105,7 +105,7 @@ struct route_table *zebra_router_get_table(struct zebra_vrf *zvrf,
 	info = XCALLOC(MTYPE_RIB_TABLE_INFO, sizeof(*info));
 	info->zvrf = zvrf;
 	info->afi = afi;
-	info->safi = SAFI_UNICAST;
```

Comment:
I have this fix and a few others already here: https://github.com/FRRouting/frr/pull/3567/commits/ea66cec4882d8e3c53cbe9465cb4313c667e87bf

I would suggest just removing this commit and taking mine instead?

Reply:
no need to take it, as you will put it.
I rebased, and removed that commit.
I just kept the single commit that fixes the netns broken case